library/src/main/kotlin/app/futured/donut/DonutProgressView.kt

```diff
@@ -449,11 +568,28 @@ class DonutProgressView @JvmOverloads constructor(
     }
 
     override fun onDraw(canvas: Canvas) {
-        super.onDraw(canvas)
         canvas.translate(centerX, centerY)
 
         bgLine.draw(canvas)
         lines.forEach { it.draw(canvas) }
+
+        if (DEBUG_COLLISIONS) {
+            drawDebugTapCollisions(canvas)
+        }
+    }
+
+    private fun drawDebugTapCollisions(canvas: Canvas) {
+        canvas.translate(-centerX, -centerY)
+        collisionRects.forEach {
+            canvas.drawRect(it.second, redPaint)
+        }
+        tapRect?.let { canvas.drawRect(it, blackPaint) }
+    }
+
+    private fun d(message: () -> String) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/a8cfaf5edea0d17a744b27747f85c2e7e1e4f6d7/library/src/main/kotlin/app/futured/donut/DonutProgressView.kt#L589"></span>Function names should be at least 3 characters long.</td>
    </tr>
  </tbody>
</table>